app/models/equipment_model.rb

```diff
@@ -121,6 +126,12 @@ def not_associated_with_self
     attachment.instance.normalized_photo_name
   end
 
+  def assign_ordering
+    update_attribute(:ordering, id)
+  end
+
+  after_create :ordering, unless: ->(em) { em.ordering }
```

Comment:
also, won't a model always have an ordering since it's a validation on both the db and model levels?